### PR TITLE
Allow match on array 

### PR DESCRIPTION
**Pull request by dvmonroe**
@beatrichartz any thoughts on this solution?  To use without having to specify the array I thought about adding a splat to the `on` method argument but that would have changed a lot of other tests.

### CODE REVIEW

**Comment by beatrichartz** [comment]
Looking good, thanks for this! Will merge once travis completes.

File: Appraisals [hunk 1]
```diff
@@ -22,6 +22,7 @@ appraise 'rails 3.2' do
   gem 'rails', '~> 3.2.0'
   gem 'jquery-rails'
   gem 'sass-rails'
+  gem 'test-unit', '~> 3.0'
```
Comment (dvmonroe) [inline Appraisals]: was getting an error on local without it

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 ENV['RAILS_ENV'] = 'test'
 ENV['BUNDLE_GEMFILE'] ||= TESTAPP_ROOT.join('Gemfile')
+ENV['RAILS_VERSION'] = ENV['BUNDLE_GEMFILE'].split('/').last
```
Comment (dvmonroe) [inline spec/spec_helper.rb]: seems hacky. Is there a better way?

**Comment by dvmonroe** [comment]
Added additional Rspec filter for rails 4 only tests in spec_helper due to previously failing tests.
Curious if there is a better way with the appraisal gem?

**Comment by jdliss** [comment]
@dvmonroe sorry for the really long wait! I made some changes to the travis config, could you merge master with this so the fixed travis config can check it?

**Comment by dvmonroe** [comment]
@jdliss no worries at all. Mind if we close this PR?  I'll open a new one with cleaner commits after I merge master.

**Comment by jdliss** [comment]
@dvmonroe yeah of course. go for it

**Comment by jdliss** [comment]
@dvmonroe Let me know when you submit that pr, ill get it merged in when travis finishes

**Pull request closed without merging**